train_management/models/abstract_dvzo_model.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django_currentuser.db.models import CurrentUserField
+
+
+class AbstractDvzoModel(models.Model):
+    create_user = CurrentUserField(related_name='created_%(class)ss')
+    create_timestamp = models.DateTimeField(_("create_timestamp"), auto_now_add=True, null=True)
```

Comment:
is this field timezone aware? How is the date time stored in the db, how is it written/read?

Reply:
It is timezone aware according to django docs. 
It saves the datetime as utc
Output is with default time zone

Reply:
So if we ever gonna change anything towards multiple timezones it will start to matter, but until then that would mean we are save that it will always be converted to utc on save and loaded with the same timezone as a timezone aware obj, right?